src/js/components/Menu.js

```diff
@@ -486,16 +486,18 @@ export default class Menu extends Component {
       );
 
       return (
-        <Button ref="control" type="icon" id={this.props.id}
-          className={classes.join(' ')}
-          tabIndex="0"
-          style={{lineHeight: this.state.controlHeight + 'px'}}
-          onClick={this._onOpen}
-          a11yTitle={menuTitle}
-          onFocus={this._onFocusControl}
-          onBlur={this._onBlurControl}>
-          {controlContents}
-        </Button>
+        <span ref="control">
```

Comment:
Now that the main container has changed one level, I believe line 255 and 309 needs to change from:

```
ReactDOM.findDOMNode(this.refs.control)
```

to 

```
ReactDOM.findDOMNode(this.refs.control).firstChild
```

Without firstChild the menu layer is not aligned with the underlying button.